build/Dockerfile-openjdk8.template

```diff
@@ -58,6 +58,15 @@ RUN chmod 755 /usr/local/bin/docker-entrypoint.sh && \
     chmod 755 /usr/local/bin/load-initdb.sh && \
     chown -R janusgraph:janusgraph ${JANUS_HOME}
 
+RUN printf "\\ndeb http://ppa.launchpad.net/rmescandon/yq/ubuntu xenial main\\n\
```

Comment:
Could you please include these instructions into the existing `RUN` statement above? If you rebase on `master`, then it already contains the installation of another package (`krb5-user`).